sirepo/sim_data/ml.py

```diff
@@ -62,3 +63,57 @@ def _lib_file_basenames(cls, data):
         if name:
             return [cls.lib_file_name_with_model_field("dataFile", "file", name)]
         return []
+
+
+class ArchiveManager:
+    import contextlib
```

Comment:
This can be a global import.

Reply:
[87b24e3](https://github.com/radiasoft/sirepo/pull/4736/commits/87b24e3f31c05fa6bb938a16525863872465d1b9)